RPGManager_development/src/rpg_database/character_sheet/interfaces/MultipleFieldsGetterSetterLanguages.java

```diff
@@ -0,0 +1,16 @@
+package rpg_database.character_sheet.interfaces;
+
+import java.util.HashSet;
+
+import rpg_database.character_sheet.Fields;
+import rpg_database.character_sheet.Languages;
+
+public interface MultipleFieldsGetterSetterLanguages<DATA, DATATYPE> extends MultipleFieldsGetterSetter{

```

Comment:
Még mindig látom. Biztos hogy törölted? A törlést is hozzá kell adnod az indexhez és pusholnod kell.